src/parser.rs

```diff
@@ -88,7 +88,6 @@ impl From<Cow<'static, str>> for Error {
 #[derive(Default)]
 pub struct ParseOptions {
     force_encoding: Option<EncodingRef>,
-    force_plural: Option<Box<Fn(u64) -> usize + 'static>>,
```

Comment:
I'd rather not remove the option of `Fn` resolver...

If I'm not mistaken, adding `+ Send + Sync` should be enough here to also allow the `Catalog` to be Send and Sync. However, closures can not be cloned yet (although the feature is coming). I would be happy to switch to simple functions `fn` though.